process/block/preprocess/sortedTransactionsProvider.go

```diff
@@ -33,7 +33,7 @@ func newAdapterTxCacheToSortedTransactionsProvider(txCache TxCache) *adapterTxCa
 
 // GetSortedTransactions gets the transactions from the cache
 func (adapter *adapterTxCacheToSortedTransactionsProvider) GetSortedTransactions() []*txcache.WrappedTransaction {
-	txs := adapter.txCache.SelectTransactionsWithBandwidth(process.MaxNumOfTxsToSelect, process.NumTxPerSenderBatchForFillingMiniblock, process.MaxGasBandwidthPerBatchPerSender)
+	txs, _ := adapter.txCache.SelectTransactions(process.TxCacheSelectionGasRequested, process.TxCacheSelectionMaxNumTxs)
```

Comment:
The returned `accumulatedGas` is ignored here, but it's logged in the mempool's codebase anyway.